### PR TITLE
hex-picker fixes

### PR DESCRIPTION
**Pull request by collinkleest**
hex-picker improvements

### CODE REVIEW

File: elements/hex-picker/hex-picker.js [hunk 1]
```diff
@@ -282,14 +312,15 @@ class HexPicker extends LitElement {
...
-      detail: value,
-    });
-    this.dispatchEvent(customEvent);
+  _dispatchChange() {
+    this.dispatchEvent(
+      new CustomEvent("value-changed", {
+        bubbles: true,
+        cancelable: false,
+        composed: false,
+        detail: { value: this },
```
Comment (btopro) [inline elements/hex-picker/hex-picker.js:321]: detail: this

Value will then be accessible to those implementing

File: elements/hex-picker/src/hex-picker.js [hunk 2]
```diff
@@ -282,14 +312,15 @@ class HexPicker extends LitElement {
...
-      detail: value,
-    });
-    this.dispatchEvent(customEvent);
+  _dispatchChange() {
+    this.dispatchEvent(
+      new CustomEvent("value-changed", {
+        bubbles: true,
+        cancelable: false,
+        composed: false,
+        detail: { value: this },
```
Comment (btopro) [inline elements/hex-picker/src/hex-picker.js:321]: Sry here, not the compiled file to fix this

**Review by btopro — CHANGES_REQUESTED** [review]
Fix event structure and this looks great

**Comment by btopro** [comment]
@collinkleest fix said 1 line and I'll accept this

File: elements/hex-picker/src/hex-picker.js (hunk 2 above)
Comment (collinkleest) [reply elements/hex-picker/src/hex-picker.js]: Fixed, I guess the PR was automatically updated when I pushed to my fork

**Pull request merged**